src/test/java/org/folio/rest/impl/OrdersImplTest.java

```diff
@@ -3774,20 +3814,10 @@ private void handleGetHoldingRecord(RoutingContext ctx) {
     private void handleGetItemRecordsFromStorage(RoutingContext ctx) {
       logger.info("handleGetItemRecordsFromStorage got: " + ctx.request().path());
 
-      try {
-        JsonObject items;
-        if (ctx.request().getParam("query").contains(PO_LINE_ID_FOR_SUCCESS_CASE)) {
```

Comment:
I think this is used to emulate case when items are partially created in inventory so the search of items by PO Line id returns 1 item all the time...

Reply:
Even after deleting this block, the tests seem to pass. Looks like we have to revisit the unit test case